src/nvim/msgpack_rpc/channel.c

```diff
@@ -251,13 +259,10 @@ Object channel_send_call(uint64_t id,
     return NIL;
```

Comment:
I would have expected a `decref()` in the `if (frame.errored) {...}` block but not here. To which `incref()` would this `decref()` correspond?